routes/controllers/user.controller.js

```diff
@@ -35,3 +36,40 @@ exports.updateRecentPc = async (req, res, next) => {
     },
   );
 };
+
+exports.postEmail = async (req, res, next) => {
+  try {
+    const transporter = nodemailer.createTransport({
+      host: "smtp.gmail.com",
+      port: 465,
+      auth: {
+        user: process.env.MAILS_EMAIL,
+        pass: process.env.MAILS_PWD,
+      },
+      secure: true,
+    });
+    const email = req.body.email;
+    const htmlContent = `
+      <h1>Hello User!</h1>
+      <p>
+        <h2>아래의 링크를 통해 다운로드 받을 수 있습니다. (차후 다운로드 링크로 변경할 예정)</h2>
+        <h3>https://github.com/Team-Saisiot/portable-trackpad-package<h3>
+      </p>
+    `;
+    const mailOption = {
+      from: process.env.MAILS_EMAIL,
+      to: email,
+      subject: "Portable Trackpad Desktop Application Download",
+      html: htmlContent,
+    };
+
+    await transporter.sendMail(mailOption);
+
+    res.json({ result: "success" });
+  } catch (error) {
+    console.error(error);
+    return res.status(500).json({
```

Comment:
```suggestion
  } catch (error) {
    console.error(error);
    
    return res.status(500).json({
```
본 코드는 return문과 위의 코드간의 행간을 1칸 띄워주는것이 좋을것같습니다.